source/stylesheets/base/_common.scss

```
@@ -31,6 +31,10 @@ button {
   font: inherit;
 }
 
+img {
+  width: 100%;
```

So small images aren't blown up and low-res, and so images that might appear elsewhere can have some base styles, let's do:

```suggestion
  display: block;
  max-width: 100%;
  margin: 0 auto $space-small;
```